### ISSUE TITLE
mattermost-preview docker container fails to start

### ISSUE THREAD

**Issue by ilyatch** [mattermost/platform#5070]
#### Summary
Latest mattermost-preview docker image for Ubuntu starts and then shuts down a short time later with an InnoDB error

#### Steps to reproduce
Follow the "preview" installation and running steps for the docker image from here: https://docs.mattermost.com/install/docker-local-machine.html#ubuntu

#### Expected behavior
Container should start and I should be able to connect to the preview server on port 8065

#### Observed behavior
What did you see happen? Please include relevant error messages and/or screenshots.
![image](https://cloud.githubusercontent.com/assets/8358856/21972179/d4f7b526-dc0b-11e6-90d3-14c0b8643592.png)

**Comment by jasonblais** [comment]
@cpanato would you be able to help?

**Comment by cpanato** [comment]
I followed the steps for ubuntu and I created a server on DigitalOcean to test. 
All went fine.

Maybe for your issue @ilyatch you will need stop the containers, remove all containers (mysql, ...)

run first:
`$docker ps`

then remove all the containers

after run
`$docker ps -a`

and check if have some mattermost containers and mysql.


just to be sure remove the images for mattermost and mysql and run the command again.
